Makefile

```diff
@@ -0,0 +1,22 @@
+.PHONY: help test install upgrade
+
+help: ## display this help message
+	@echo "Please use \`make <target>' where <target> is one of"
+	@perl -nle'print $& if m{^[a-zA-Z_-]+:.*?## .*$$}' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m  %-25s\033[0m %s\n", $$1, $$2}'
+
+install: ## install the flow-control xblock
+	pip install .
+
+requirements: ## fetch development requirements
+	pip install -r requirements/dev.txt
+
+test: ## test using tox
+	pip install tox
+	tox	
+
+upgrade: export CUSTOM_COMPILE_COMMAND=make upgrade
+upgrade: ## update the pip requirements files to use the latest releases satisfying our constraints
+	pip install -q pip-tools
```

Comment:
Please install this using the requirements/pip-tools.txt file generated by `make upgrade`

Reply:
Edited accordingly.